tests/system/test_requests.py

```diff
@@ -14,7 +14,6 @@
     from io import StringIO
 
 
-@integration_test
```

Comment:
why did you remove the annotations?

Reply:
These tests do not require Elasticsearch or Kibana, therefore they do not need the annotation.

Reply:
I must be missing something. Don't they all use Elasticsearch by virtue of inheriting from ServerBaseTest and ClientSideBaseTest? Or will they pass even if Elasticsearch isn't running?

Reply:
`Elasticsearch` is only created in `ElasticTest` which inherits from `ServerBaseTest`. So everything inheriting from `ElasticTest` and `ClientSideElasticTest` needs to be marked `integration_test`, while others don't.